### PR TITLE
Unicode fixes

### PR DESCRIPTION
**Pull request by samoneilll**

### CODE REVIEW

File: js/app/util.js [hunk 1]
```diff
@@ -951,13 +951,10 @@ define([
...
-    let unicodeToString = (text) => {
-      let result = text.replace(/\\u[\dA-F]{4}/gi,
-        function (match) {
-          return String.fromCharCode(parseInt(match.replace(/\\u/g, ''), 16));
-      });
-      return result.substring(0, 2) == "u'" ? result.substring(2, result.length - 1) : result;
-    };
+    const unicodeToString = (text) => {
+        const result = text.replace(/\\u[\dA-F]{4}/gi, (match) => String.fromCharCode(parseInt(match.replace(/\\u/g, ''), 16)))
+        return result.substring(0, 2) == "u'" ? result.substring(2, result.length - 1) : result;
```
Comment (danbfield) [inline js/app/util.js]: You can remove that `;` at the end. You don't need them. I made a mistake when I copied it over from his original code!

Comment (samoneilll) [reply js/app/util.js]: done ✝

**Review by danbfield — APPROVED** [review]
Ship it

**Comment by TyrHeimdalEVE** [comment]
@samoneilll Is this ready to be merged for next version?

**Pull request merged**